Cargo.toml

```diff
@@ -46,11 +46,11 @@ embedded-io-async = "0.6.1"
 embedded-services = { path = "./embedded-service" }
 embedded-storage = "0.3"
 embedded-storage-async = "0.4.1"
-embedded-usb-pd = { git = "https://github.com/OpenDevicePartnership/embedded-usb-pd", default-features = false }
+embedded-usb-pd = { path = "../embedded-usb-pd", default-features = false }
```

Comment:
The `[patch]` table is often recommended for local development rather than changing the `[dependencies]` table.